Woof/Woof/View/ReusableViews/LabelWithTextAndImage.swift

```diff
@@ -0,0 +1,25 @@
+import SwiftUI
+
+/// A view displaying an icon and the portion of text.
+struct LabelWithTextAndImage: View {
+    /// The name of the icon that the label displays.
+    let iconName: String
+
+    /// The text that the label displays.
+    let text: String
+
+    var body: some View {
+        Label(text, systemImage: iconName)
+            .font(.system(size: AppStyle.FontStyle.body.size, weight: .bold))
+            .background(Color.App.purpleLight)
+            .foregroundColor(Color.App.white)
+    }
+}
+
```

Comment:
Do we have to create the view at all?
Why don't use the default label and specify foreground and background color in the parent view?

Reply:
I thought , it will make the code more compact and avoid  repeating the same lines of code.

Reply:
Agree, and removed unnecessary extra view.